src/AppMap.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import Leaflet from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import {
+  Map,
+  Polyline,
+  TileLayer,
+  Marker,
+  Popup
+} from 'react-leaflet';
+import '../map.css';
+import Spinner from 'react-bootstrap/Spinner';
+import { NoPaths, getData } from '../Shared';
+import Tab from 'react-bootstrap/Tab';
+import Tabs from 'react-bootstrap/Tabs';
+
+// pull images from cdn instead of storing locally
+Leaflet.Icon.Default.imagePath = '//cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.4/images/';
+
+export default class AppMap extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {markerData: [], isLoading: true};
+	}
+
+	async componentWillMount() {
+		// urls for the direction jsons
+		var marker_urls;
+		const names = await getData('https://github.com/SASE-Labs-2021/gopher-a-bite/api/restNames.json');
+		marker_urls = Object.values(names).map(name => 'https://github.com/SASE-Labs-2021/gopher-a-bite/api/popUpInfo/' + name + '.json');
```

Comment:
This line won't work. You'd need to either activate GitHub pages and change the link or add '?raw=true', both of which are bad practices. Instead, we should add a route to the API that looks at the restaurant name to file name map and returns the corresponding JSON.